meetings/2022-04-25.md

```diff
@@ -0,0 +1,76 @@
+
+## Present
+* Aaron Coburn
+* Nicolas AS
+* Elf Pavlik
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md)
+* If this is your first time, welcome! please introduce yourself.
+
+### Scribes
+* Nicolas AS
+
+---
+
+## Agenda
+
+### Issues
+
+https://github.com/solid/solid-oidc/issues
+
+* Prioritization
+* Timeframe
+* [Version format](https://github.com/solid/solid-oidc/issues/114) (e.g., 0.2)
+
+## Minutes
+
+### Mutliple open editorial issues
+
+* AC: The 0.1 version has been released, which is a huge milestone. We just need to make sure we are aligned with the editors regarding version numbers. Take IETF for instance, with draft-v01, draft-v02... iterated until formal recognition. Currently, the draft is published under the TR label, which may lead to expect more stability than the stage we are in, we want to be extra clear about our assumptions and expectations.
+* eP: Once we've iterated through enough 0.x versions and feel ready, we can move to the CR status.
+* AC: In our draft, we can add a PR to update the version to Editor's draft 0.2, clearly stating that's the future version for the TR draft.
+* eP: Is it required to call out editor's draft, as we're not under the /TR ?
+* AC: Aligning the version number between the tag and the text makes it easier to work with in automation.
+* eP: What you think is most practical with automation should be good.
+* eP: I can go through the open issues about the primer.
+* AC: A bunch of these issues should be individually small size, so if we could close some up that would be good. To prioritize, we can first focus on issues touching on normative text, then those who deal with terminology/abstract concepts (non-normative). Happy to take some of these.
+* eP: Some of these issue may come from a lack of familiarity with OAuth. It could be settled with a simple reference to the underlying spec defining the term.
+* AC: In [that particular case](https://github.com/solid/solid-oidc/issues/139), on the first instance change "code" into OAuth 2.0 code linking tho the auth code definition.
+* eP: We can provide a reference, but don't need to get into too much details because implementors will have familiarity with OIDC/Oauth
```

Comment:
```suggestion
* eP: We can provide a reference, but don't need to get into too much detail because implementors will have familiarity with OIDC/Oauth
```